src/ui/public/vislib/lib/layout/layout_types.js

```diff
@@ -17,6 +17,7 @@ export default function LayoutTypeFactory(Private) {
     line: Private(VislibLibLayoutTypesColumnLayoutProvider),
```

Comment:
Is anything using the line, area, etc. layouts anymore? It's all point_series now right?